tests/test_throttled_http_client.py

```diff
@@ -82,9 +70,10 @@ def test_pickled_minimal_response_should_contain_signature(self):
             status_code=200, headers={}, text="foo")))
 
     def test_throttled_http_client_base_response_should_tolerate_headerless_response(self):
-        http_client = ThrottledHttpClientBase(DummyHttpClientWithoutResponseHeaders(
-            status_code=200, response_text="foo"))
-        response = http_client.post("https://example.com")
+        http_client = DummyHttpClient(status_code=200, response_text="foo")
```

Comment:
should we add the regression bug info here?

Reply:
OK, I'll probably add some inline comment in my next commit.